ops/packer/windows/windows.pkr.hcl

```diff
@@ -0,0 +1,90 @@
+packer {
```

Comment:
any recommended reading for this? (along with the linux version).

Reply:
Sure, start with

* https://runs-on.com/guides/building-custom-ami-with-packer/
* https://developer.hashicorp.com/packer/tutorials/aws-get-started/aws-get-started-build-image